ngx-nuts-and-bolts-docs/docs/loading-state/loading-state.md

```diff
@@ -0,0 +1,179 @@
+---
+id: component-with-loading-state
+title: Component with loading state
+sidebar_label: Component with loading state
+---
+
+# Component with loading state
```

Comment:
Remember to update this after #17 is merged because this readme file was updated.

Reply:
Sure, this was only for testing purposes